clients/clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -257,7 +257,8 @@ interface ClearlyDefinedService {
 
     /**
      * Get information about the harvest tools that have produced data for the component described by [type],
-     * [provider], [namespace] (use "-" if not applicable), [name], and [revision], see
+     * [provider], [namespace] (use "-" if not applicable), [name], and [revision] (when empty, the latest revision will
```

Comment:
I believe the docs should also state what the behavior is, when revision is empty and it doesn't make sense for the provider to query the latest revision.

Reply:
This sentence just comes as-is from the [official docs](https://api.clearlydefined.io/api-docs/#/harvest/get_harvest__type___provider___namespace___name___revision_) that also don't state what happens if an empty revision is used with a provider that does not support it, so I can't tell.

Reply:
I see, thanks.